pkg/trace/operation.go

```diff
@@ -122,6 +122,14 @@ func (o *Operation) Errorf(format string, args ...interface{}) {
 	Logger.Errorf("%s: %s", o.header(), fmt.Sprintf(format, args...))
 }
 
+func (o *Operation) Warnf(format string, args ...interface{}) {
+	Logger.Warnf("%s: %s", o.header(), fmt.Sprintf(format, args...))
+}
+
+func (o *Operation) SPrintf(format string, args ...interface{}) string {
```

Comment:
Yeah, I found a use for it though when passing it to things like `trace.Begin`. If we ever want to make a string with the operation ID in it, this would be the simplest approach.